rllib/models/tf/visionnet.py

```diff
@@ -59,12 +63,20 @@ def __init__(self, obs_space, action_space, num_outputs, model_config,
                 padding="valid",
                 data_format="channels_last",
                 name="conv{}".format(i + 1))(last_layer)
-            conv_out = tf.keras.layers.Conv2D(
-                num_outputs, [1, 1],
-                activation=None,
-                padding="same",
-                data_format="channels_last",
-                name="conv_out")(last_layer)
+
+            if num_outputs:
+                conv_out = tf.keras.layers.Conv2D(
+                    num_outputs,
+                    [1, 1],
+                    activation=None,
+                    padding="same",
+                    data_format="channels_last",
+                    name="conv_out")(last_layer)
+            else:
```

Comment:
If `num_outputs` is None (e.g. when wrapping the Conv2D with a custom or LSTM Model), the last layer will be a convenient flatten one so we can simply connect the wrapper to this Conv2D Model.

self.num_outputs will then be set to the number of nodes in this flatten layer (convenience to the wrapping Model).